src/main/java/com/project/store/exceptions/category/ProductCategoryExceptionMessages.java

```diff
@@ -0,0 +1,24 @@
+package com.project.store.exceptions.category;
+
+import org.springframework.http.HttpStatus;
+
+public enum ProductCategoryExceptionMessages {
+
+    PRODUCT_CATEGORY_NOT_FOUND("Product Category not found!", HttpStatus.NOT_FOUND.toString());
```

Comment:
I would use only one enum for all the errors, so I would merge this one with the one created for Customer

Reply:
Exception messages from Customer and ProductCategories were merged into a single enum called Exception messages